src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -762,7 +754,6 @@ void LedgerConsensusImp::statePreClose ()
         sinceClose = 1000 * (
             app_.timeKeeper().closeTime().time_since_epoch().count()
             - consensus_.getLastCloseTime ());
-        idleInterval = LEDGER_IDLE_INTERVAL;
     }
 
     idleInterval = std::max (idleInterval, LEDGER_IDLE_INTERVAL);
```

Comment:
Since you're no longer setting `idleInterval` above this line, except to initialize it, it's always `0` here. In that case, you could move the declaration and initialization down here: ```int idleInterval = LEDGER_IDLE_INTERVAL;```